README.md

```diff
@@ -6,7 +6,91 @@ For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/maste
+* https://github.com/nodejs/http-parser — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/LTS — administration delegated to the LTS Working Group
+* https://github.com/nodejs/node-eps — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/nodejs-dist-indexer — administration delegated to the Build Working Group
+* https://github.com/nodejs/nodejs-latest-linker — administration delegated to the Build Working Group
+* https://github.com/nodejs/nodejs-nightly-builder — administration delegated to the Build Working Group
+* https://github.com/nodejs/nodejs.org — administration delegated to the Website Working Group
+* https://github.com/nodejs/secrets — _private_, administration delegated to the Build Working Group
+* https://github.com/libuv/libuv — currently under Incubation
+
+### Adjacent Projects
+
+The TSC is responsible for a number of projects that are not strictly required to plan, test, build, document and ship Node.js releases. Projects that are _adjacent_ are either created from within the technical organization managed by the TSC or are adopted into that organization from outside.
+
+In the case of adopting existing projects, once the TSC has decided that adoption appropriate, it should seek agreement from the Node.js Foundation Board for such adoption as it impacts on the scope of technical activities of the Foundation.
+
+Current adjacent projects that are within the TSC's scope of responsibility can be found at the following locations:
+
+* https://github.com/nodejs/api — administration delegated to the API Working Group
+* https://github.com/nodejs/benchmarking — administration delegated to the Benchmarking Working Group
+* https://github.com/nodejs/code-and-learn
+* https://github.com/nodejs/diagnostics — administration delegated to the Diagnostics Working Group
+* https://github.com/nodejs/docs — administration delegated to the Documentation Working Group
+* https://github.com/nodejs/education
+* https://github.com/nodejs/email — administration delegated to the Build Working Group
+* https://github.com/nodejs/evangelism — administration delegated to the Evangelism Working Group
+* https://github.com/nodejs/github-bot — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/help — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/http — administration delegated to the HTTP Working Group
+* https://github.com/nodejs/http2 — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/iojs.org — administration delegated to the Website Working Group
+* https://github.com/nodejs/installer — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/Intl — administration delegated to the Intl Working Group
+* https://github.com/nodejs/llnode — administration delegated to the Core Technical Committee
```

Comment:
Can y'all clarify this please, /cc @indutny 
I see it in GOVERNANCE.md (LICENSE.md says nodereport btw) but the GitHub permissions suggest that it's its own group. Should we fix GitHub to be nodejs/port-mortem? Or are there extra people involved in this that aren't in post-mortem?